R/plotFeatureLoadings.R

```diff
@@ -0,0 +1,404 @@
+#' Plot tree with feature loadings after performing a reducedDim (e.g. PCA, LDA, NMF)
+#'
+#' Based on the \code{\link[diffTop:plotASVcircular]{plotASVcircular}} method using phyloseq 
+#' and has been converted to use TreeSummarizedExperiment objects.
+#'
+#' @param object a
+#'   \code{\link[TreeSummarizedExperiment:TreeSummarizedExperiment-class]{TreeSummarizedExperiment}}
+#'   object.
+#' 
+#' @param method One dimensionality reduction method of \code{c("PCA", "LDA", "NMF")}
+#'   indicating which method will be used.
+#'   (default: \code{method = "PCA"})
+#'  
+#' @param rank a single character defining a taxonomic rank. Must be a value of
+#'   \code{taxonomyRanks()} function.
+#'   
+#' @param plot One way to plot feature loadings of \code{c("Heatmap", "Barplot", "Screeplot", "Tree")} 
+#'   (default: \code{plot = "Heatmap"})
+#'   
+#' @details
+#' # TO DO 
+#' @return 
+#' A \code{ggplot2} object. A circular plot annotated with TreeSummarizedExperiment object.
+#'
+#' @name plotFeatureLoadings
```

Comment:
I think the @name tag is not needed (?). 

The function name could be shorter, just plotLoadings?